AVFoundation-Combine/Extensions/AVPlayerItem+Publishers.swift

```diff
@@ -37,4 +37,14 @@ public extension AVPlayerItem {
     func durationPublisher() -> AnyPublisher<CMTime, Never> {
         publisher(for: \.duration).eraseToAnyPublisher()
     }
+    
+    /// Publisher that wraps `Notification.Name.AVPlayerItemDidPlayToEndTime`
+    ///
+    /// - Returns: Publisher that emits a value  when the item plays to its end time.
+    func didPlayToEndPublisher(_ notificationCenter: NotificationCenter = .default) -> AnyPublisher<Void, Never> {
```

Comment:
```suggestion
    func didPlayToEndTimePublisher(_ notificationCenter: NotificationCenter = .default) -> AnyPublisher<Void, Never> {
```

Reply:
Here, i think it's important to mirror as much as possible the name of the original property being published

Reply:
It that's the case, we could also change the signature from `<Void, Never>` to `<Notification, Never>` and pass along the original notification. What do you think about that?

Reply:
Yes, absolutely for completeness it should be that way (even if in our example it doesn't make a difference). In a certain Keynote presentation you might notice this is the original way this `Publisher` had been imagined ;)